src/index.js

```diff
@@ -15,28 +16,47 @@ class App extends React.Component {
     this.removeFromOrder = this.removeFromOrder.bind(this);
   }
 
-  addToOrder(name, price, bread, isHot) {
-    this.setState(prevState => ({
-      orderItems: [...prevState.orderItems, { name, price, bread, isHot }],
-      orderTotal: prevState.orderTotal + price
-    }));
-  }
-
-  removeFromOrder(i) {
-    this.setState(prevState => ({
-      orderItems: prevState.orderItems.filter((_, idx) => idx !== i)
-    }));
-    this.setState(prevState => {
+  componentDidMount() {
+    const orderRef = firebase.database().ref('order');
+    orderRef.on('value', (snapshot) => {
+      const items = snapshot.exists() ? snapshot.val() : {};
+      const orderItems = [];
       let orderTotal = 0;
-      prevState.orderItems.forEach(order => {
-        orderTotal += order.price;
+
+      Object.keys(items).forEach((key) => {
+        orderItems.push({
```

Comment:
You can destructure here again man so your push is cleaner with shorthand.

`const { id, bread, isHot, name, price } = items[key]`